pintc/src/error/handler.rs

```diff
@@ -12,10 +12,10 @@ pub struct Handler {
 /// Contains the actual data for `Handler`.
 /// Modelled this way to afford an API using interior mutability.
 #[derive(Default, Debug)]
-struct HandlerInner {
-    /// The sink through which errors will be emitted.
-    errors: Vec<Error>,
-    // TODO: add warnings here
+pub struct HandlerInner {
+    /// The sink through which errors and warnings will be emitted.
+    pub errors: Vec<Error>,
+    pub warnings: Vec<Warning>,
```

Comment:
Same with these. We shouldn't have access to `errors` and `warnings` directly like this. We have the API below to access and manipulate these.